src/main/java/chess/domain/ChessGame.java

```diff
@@ -0,0 +1,83 @@
+package chess.domain;
+
+import chess.domain.position.File;
+import chess.domain.position.Position;
+import chess.domain.position.Rank;
+import java.util.List;
+import java.util.Objects;
+
+public class ChessGame {
+
+    private static final int COMMAND_INDEX = 0;
+    private static final int SOURCE_POSITION_INDEX = 1;
+    private static final int DEST_POSITION_INDEX = 2;
+    private static final int FILE_RANK_DIVIDING_INDEX = 1;
+
+    private final ChessBoard chessBoard;
+    private GameState state;
+
+
+    private ChessGame(ChessBoard chessBoard, GameState state) {
+        this.chessBoard = chessBoard;
+        this.state = state;
+    }
+
+    public static ChessGame startNewGame(final String command) {
+        validateStartCommand(command);
+        ChessBoard chessBoard = ChessBoardFactory.create();
+        return new ChessGame(chessBoard, GameState.RUNNING);
+    }
+
+    private static void validateStartCommand(final String command) {
+        if (Objects.isNull(command) || !(Command.from(command) == Command.START)) {
```

Comment:
```suggestion
        if (Objects.isNull(command) || Command.from(command) != Command.START) {
```
이렇게 쓸수 있을듯 하네요.

어느것이 더 가독성이 좋은지는 선택일듯 해요 ㅎㅎ